src/Yasumi/Provider/Bosnia.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * This file is part of the Yasumi package.
+ *
+ * Copyright (c) 2015 - 2018 AzuyaLabs
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @author Sacha Telgenhof <stelgenhof@gmail.com>
+ */
+
+namespace Yasumi\Provider;
+
+use DateTime;
+use DateTimeZone;
+use Yasumi\Holiday;
+
+/**
+ * Provider for all holidays in Croatia.
```

Comment:
You might want to correct this :) Please also check everywhere else in all the files if there is anything that needs to be changed.